### DISCUSSION TITLE
Docker image 1.5.4 missing "latest" tag

### DISCUSSION THREAD

**Discussion by tfl-jb** [ltb-project/self-service-password#820] [Q&A] [answered]
I wanted to update to 1.5.4 and found, that the image is missing the tag "latest". Is that on purpose or a mistake? 

I read on an old issue (https://github.com/ltb-project/self-service-password/issues/616), that the master branch could be "work in progress". So, I'm not sure if I'm on the right track.

**Comment by coudot** [comment, marked as answer]
Hello @tfl-jb 

I pushed the new tags, please let me know if this is ok

**Reply by tfl-jb** [reply]
Aweseome, thanks for the fast update.
